### PR TITLE
Bump packages, require node 14

### PR DESCRIPTION
**Pull request by rbruckheimer**
## Description

Bumped dependencies to get rid of vulnerabilities and set minimum node version to 14 to match mocha 10.0

## Proposed Changes

  -
  -

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,9 +24,9 @@
   },
   "devDependencies": {
     "@types/argv": "0.0.5",
-    "@types/node": "^16.4.10",
-    "@types/lodash": "^4.14.172",
-    "chai": "^4.3.4",
-    "mocha": "^9.0.3"
+    "@types/node": "^18.0.0",
```
Comment (RMcNeely) [inline package.json:27]: Does the `@types/node` package reflect the definitons for node 18? Also is it really something we **_need_** to include?

Comment (rbruckheimer) [reply package.json:27]: I would assume so.  Do you think we should purposely not bump from 16, so no one sees any types specific to 18?  Should we restrict the engine to node 14 and 16 only?

Comment (RMcNeely) [reply package.json:27]: I guess I'm wondering what the point of updating the types definitions is and if there's a reason to require something newer than the engine. Did this trigger an audit warning or are we just upgrading it while we were upgrading mocha and chai?

Comment (rbruckheimer) [reply package.json:27]: It did not trigger an update warning, it was just the most recent version available.  As it was set to 16.x before and we were allowing node 12, I figured it wasn't an issue using the types from 18.

Comment (RMcNeely) [reply package.json:27]: alright if it wasn't an explict decision then I don't see it doing any harm, just trying to figure out why it got updated.

**Review by RMcNeely — APPROVED** [review]

**Pull request merged**